### ISSUE TITLE
ChimeraOS error: "Read Only -- Command exited with code 256"

### ISSUE THREAD

**Issue by jim-80** [moraroy/NonSteamLaunchers-On-Steam-Deck#173]
[NonSteamLaunchers-install.log](https://github.com/moraroy/NonSteamLaunchers-On-Steam-Deck/files/14013981/NonSteamLaunchers-install.log)

I've been using your tool on SteamOS and I wanted to also use it on a ChimeraOS handheld. I know this is supposed to be a Steam Deck script, so this isn't an issue as such, but it would be awesome to get it working on Chimera!

The downloads seem to happen fine, then the script closes Steam and re-launches it. Just before the re-launch, the terminal window throws an error:

`Read Only -- Command exited with code 256`

And the script dies.

I've attached my log. Any idea on this one?

**Comment by jim-80** [comment]
thought: it might be because /home/deck/ is hard-coded in a few paths. That'll be different on Chimera. I'll do a find/replace and test the theory tomorrow.

**Comment by moraroy** [comment]
@jim-80 yeah I noticed this yesterday I'll be upading one without "/home/deck" soon

**Comment by moraroy** [comment]
@jim-80 just updated and got rid of all /home/deck/ let me know if yous till get this error

**Comment by jim-80** [comment]
@moraroy Thanks for doing this. I've just been testing. The script does seem to die at the same point and with the same error, but there's no popup/exception now and the selected installs do get added to Steam. 

- Downloads happen properly
- Shortcuts properly created in Steam
- Download folder successfully deletes
- The script successfully detects the launchers it has already downloaded/installed (but not for the browser shortcuts), so those appear in the index if you re-run it
- Uninstall works
- It does NOT scrape artwork for the Steam shortcuts, so that's the one manual step left

So it's a bit messy with the script exit, and there's one manual bit, but it's running pretty well on ChimeraOS :)

**Comment by moraroy** [comment]
@jim-80 you're modules aren't getting downloaded thats why you're not in seeing the artwork. The error is in the .py file, I think I'm using HOME as a path? You may have to check the file and see if that path is or the way the path is made is compatable with chimera? Maybe I can change it to ${logged_in_home} like I've been doing? Haha idk
